src/pages/getting-started/index.md

```diff
@@ -11,19 +11,26 @@ The first step in accessing the Photoshop APIs is getting authenticated. For tha
 ## Get access
 Here are the steps to start with our API
```

Comment:
"Here are the steps top get started with the Ps API." 

Lets change this sentence for better grammar and clarity.